teps/0115-tekton-catalog-git-based-versioning.md

```diff
@@ -605,7 +616,7 @@ spec:
 
 #### Hub Resolver
 
-[Hub Resolver][hub-resolver] has a `"version"` parameter that can be used to get a specific version of a resource. The `"version"` should continue to use the simple version of [semver][semver] (i.e. `<major-version>.<minor-version>`). For example [this version][bp-hub] of buildpacks `Task`:
+[Hub Resolver][hub-resolver] has a `"version"` parameter that can be used to get a specific version of a resource. For example [this version][bp-hub] of buildpacks `Task`:
```

Comment:
Curious if Hub resolver would accept different version formats?

Reply:
We currently don't have version format validation in the Hub Resolver, and we will call the Artifact Hub API with the version as it is (except for appending 0 as patch version if not provided).